src/olympia/abuse/models.py

```diff
@@ -171,6 +186,8 @@ class AbuseReport(ModelBase):
     state = models.PositiveSmallIntegerField(
         default=STATES.UNTRIAGED, choices=STATES.choices
     )
+    # if this report has been submitted to cinder this is the job reference there
+    cinder_job_id = models.CharField(max_length=36, default=None, null=True)
```

Comment:
Not sure this should live in this model. In the issue I mentioned adding a `AbuseReportDecision` but maybe this should be `AbuseReportCinder` or whatever and it would hold everything related to Cinder ?

Reply:
https://github.com/mozilla/addons-server/pull/21260/commits/07227894165ae930a0dcb2433c7e1b90e8af71c3 adds a `CinderReport` model.